### PR TITLE
Python entry points

### PR DESCRIPTION
**Pull request by robinandeer**
Adding TIL on how to use "entry points" for defining command line interfaces in Python.

### CODE REVIEW

File: python/setuptools_entry_points.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Setuptools entry points
+The most common way of exposing a Python CLI is to use the "scripts"
+keyword in ``setup.py``.
```
Comment (JoelQ) [inline python/setuptools_entry_points.md]: Thoughts on explicitly stating that this is for building a Python package?

Comment (robinandeer) [inline python/setuptools_entry_points.md]: That's true - I could develop that sentence a bit. Would something as simple as this work?

> When building a Python package, the most common way of exposing a command line interface (CLI)...

**Comment by JoelQ** [comment]
Thanks for sharing @robinandeer ! I've merged this as 156f6aad6ba711e28c4c6c3ec7ea39b93ad0edae . I'd like to credit you when we tweet this learning, what is your twitter handle?

**Comment by robinandeer** [comment]
Oh fantastic, thanks! My twitter handle is: @robinandeer (same as GH)

**Pull request closed without merging**